### ISSUE TITLE
Unclear commit messages

### ISSUE THREAD

**Issue by yvonne-liu** [facn2/ApolloXII#49]
Try to steer clear from vague commit messages like:

"fix somestuff in css"
"Made some small changes to router"
"changed css and handler"
"edit html and add images"

You get the idea :blush:

**Comment by yvonne-liu** [comment]
Oh, the same applies to your PRs as well :wink:

**Comment by veredrec** [comment]
@yvonne-liu yes, definitely something that we would try to be more aware of! Thanks!
